src/logdb/test/logdb_tests.c

```diff
@@ -120,7 +120,7 @@ void test_logdb(logdb_log_db* (*new_func)())
     u_assert_int_eq(memcmp(value_test->str, value->str, value->len), 0);
 
     value_test = logdb_find(db, key2);
-    u_assert_int_eq((int)value_test, 0); /* should be null */
+    u_assert_uint32_eq((uintptr_t)value_test, 0); /* should be null */
```

Comment:
Wait I had an idea. This fixes the warning, it's simpler, and should work with any size of pointer:  
u_assert_int_eq(!value_test, 1);